README.md

```
@@ -67,7 +68,8 @@ for more details.
 
 ### Development version
 
-The latest development version of the installer from `develop` branch can be found at https://vortex-docs.netlify.app/install
+The latest development version of the installer from `develop` branch can be
+found at https://vortex-docs.netlify.app/install
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)